webapp/src/pages/AdminPage/AdminPage.tsx

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
This page is a draft to facilitate testing. The final shape should be nothing like this.